src/desktop/apps/artist2/server.js

```diff
@@ -1,15 +1,15 @@
-import { buildServerApp } from 'reaction/Artsy/Router'
-import { Meta, query, toJSONLD } from './components/Meta'
-import { renderLayout } from '@artsy/stitch'
-import { routes } from 'reaction/Apps/Artist/routes'
-import express from 'express'
-import metaphysics from 'lib/metaphysics.coffee'
-import React from 'react'
-import styled from 'styled-components'
+import { buildServerApp } from "reaction/Artsy/Router/server"
```

Comment:
@alloy - so this was the culprit? Makes sense (oversight)

Reply:
Yup, this was it!